plugins/candela/web_client/templates/configView.pug

```diff
@@ -6,5 +6,7 @@ p
   |  to render table files directly in Girder.
 
 p.
-  This plugin requires no configuration. Simply upload a CSV file as an item,
-  then set the Candela visualization type and options.
+  This plugin requires no configuration. Simply upload a CSV file as an item
```

Comment:
I'm not sure why we expose a configuration option for this plugin at all. It seems like a lot of pointless code complexity just to document something.

I think that:
* all of the code for the Candella config page should be removed
* the text should be moved to a new section of `docs/plugins.rst`
* the line `url: "http://girder.readthedocs.io/en/latest/plugins.html#candella"` should be added to `plugins/candella/plugin.json`.

This whole process should also be done for the `vega` plugin.

If you find my suggestion to be outside the scope of this PR, I'd be happy to implement it myself in another PR.